calabash-cucumber/scripts/.irbrc

```diff
@@ -47,3 +43,22 @@ extend Calabash::Cucumber::Operations
 def embed(x,y=nil,z=nil)
   puts "Screenshot at #{x}"
 end
+
+require "calabash-cucumber"
+
+def preferences
+  @preferences ||= Calabash::Cucumber::Preferences.new
+end
+
+def disable_usage_tracking
+  preferences.usage_tracking = "none"
+  puts "Calabash will not collect usage information."
+  "none"
+end
+
+def enable_usage_tracking(level="system_info")
+  preferences.usage_tracking = level
```

Comment:
Is there a threading concern here since this is writing to disk? I don't know how Ruby handles `io` by default, but would it perhaps make sense to put the preferences `io` into a serial queue?